src/80850_len_3060.c

```diff
@@ -506,30 +506,43 @@ void status_menu_draw_stat(s32 id, s32 x, s32 y, s32 currentValue, s32 maxValue)
     }
 }
 
-#ifdef NON_MATCHING
 //regalloc issues
+#ifdef NON_MATCHING
 void update_status_menu(void) {
     UiStatus* uiStatus = &gUIStatus;
     PlayerData* playerData = &gPlayerData;
     PlayerStatus* playerStatus = &gPlayerStatus;
-    s8 new_var;
     s32 s0;
     s32 s02;
     s32 fp;
+    s32 new_var;
     s32 s5;
     s32 s6;
     s32 id;
     s32 s4;
+    s32 s44;
     s32 s7;
     s32 s1;
+    s32 s66;
     s32 v0;
     s32 temp;
-s32 new_var2;
+
     s32 sp50;
     s32 sp54;
     s32 sp5C;
     s32 sp60;
-    s32 something;
+
+    /*
+        regalloc:
+        target  |  current     |  do while
+        ------------------------------------------
+        s5      |  sp54        |   s7
+        s6      |  s5          |   s5
+        s7      |  s6          |   s6
+        s8      |  sp50        |   sp50
+        sp50    |  s8          |   sp54
+        sp54    |  s7          |   s8
+    */
```

Comment:
Probably can remove this now its matched?

Reply:
that one isn't matched - it's another one. the do while 0 stuff is wip work on that function that unnunu found I think :)